openedx/core/djangoapps/catalog/utils.py

```diff
@@ -151,16 +173,108 @@ def get_program_types():
         return []
 
 
-def get_programs_with_type_logo():
+def _get_program_instructors(program):
```

Comment:
I see, the catalog currently only pulls this data from the Drupal marketing site. We don't know how long it'll be until full configuration of a course via the catalog is supported; I suspect it won't be soon. The catalog data loaders already have an established pattern for dealing with LMSes that don't have a Drupal marketing site. Updating them to pull in this instructor data wouldn't be hard, and I urge you to reconsider making the change. If you do, you can just use staff from the response don't need to code being introduced here.

If what's here stays, I'm concerned about direct use of the modulestore. Accessing the modulestore is expensive. Have you considered adding instructor information to CourseOverviews? That aside, this module is only meant to contain utilities for retrieving data directly from the catalog. This stitching together of that data with other data from the modulestore seems out of place here.